src/index.ts

```diff
@@ -32,6 +32,13 @@ export interface ECRDeploymentProps {
    */
   readonly dest: IImageName;
 
+  /**
+   * The image architecture to be copied.
+   *
+   * @default - ""
```

Comment:
please explain what this default means in the doc string

Reply:
Updated